ckanext/ckanext-apicatalog/ckanext/apicatalog/plugin.py

```diff
@@ -931,9 +929,6 @@ def after_show(self, context, data_dict):
         if context.get('ignore_auth') or (context.get('auth_user_obj') and context.get('auth_user_obj').sysadmin):
             return data_dict
 
-        if data_dict.get('xroad_removed') is True:
```

Comment:
This probably is still required as permission labels do not affect direct urls.

Reply:
The direct URL gives a permission error, as reflected in the test change.